tests/Unit/inc/Engine/Media/Lazyload/CSS/Front/Extractor/extract.php

```diff
@@ -7,6 +7,7 @@
 
 use WP_Rocket\Tests\Unit\TestCase;
 use Brain\Monkey\Functions;
+
```

Comment:
Is this change intended ? As there is nothing else in this test that has been changed, it seems like committed noise